package.json

```diff
@@ -10,19 +10,26 @@
   "umd:main": "dist/redux-on-action.umd.js",
   "scripts": {
     "build": "microbundle src/index.js -o dist/redux-on-action.js",
-    "test": "eslint src",
-    "release": "npm run build && npm t && git commit -am $npm_package_version && git tag $npm_package_version && git push && git push --tags && npm publish"
+    "lint": "eslint src",
+    "test": "jest",
+    "release": "npm run lint && npm run build && npm t && git commit -am $npm_package_version && git tag $npm_package_version && git push && git push --tags && npm publish"
   },
   "devDependencies": {
+    "@babel/core": "7.1.2",
+    "@babel/preset-env": "7.1.0",
+    "babel-core": "7.0.0-bridge.0",
+    "babel-jest": "23.6.0",
     "eslint": "5.5.0",
     "eslint-config-airbnb-base": "13.1.0",
     "eslint-plugin-import": "2.14.0",
-    "microbundle": "0.6.0"
+    "jest": "23.6.0",
+    "microbundle": "0.6.0",
+    "regenerator-runtime": "0.12.1"
```

Comment:
Same thing - why is this necessary?

Reply:
But for this one, I'm not really sure ... [The guide on using Babel on Jest](https://jestjs.io/docs/en/getting-started#using-babel) said to also explicitly install `regenerator-runtime`. I'll dig deeper on it later on.

Reply:
I removed it and things seemed to worked fine. I guess it was there so that we can write test in async/await syntax, but since we're not using it, I removed it from dependency

Reply:
`async/await` is available natively in node so I don't think so. The only thing that comes to my mind in terms of transpilation is ES2015+ import/export syntax.

Reply:
I'll take a look at this. Sorry about me being so quiet for a long time I'm kind of busy right now. I'll find some time to look at this.

Thank you for your effort! Really appreciate this! 😃